src/llvm-simdloop.cpp

```diff
@@ -203,8 +203,9 @@ bool LowerSIMDLoop::markSIMDLoop(Module &M, Function *marker, bool ivdep)
         MDNode *n = L->getLoopID();
         if (!n) {
             // Loop does not have a LoopID yet, so give it one.
-            n = MDNode::get(Lh->getContext(), ArrayRef<Metadata *>(NULL));
-            n->replaceOperandWith(0, n);
+            auto temp_n = MDNode::getTemporary(Lh->getContext(), ArrayRef<Metadata *>(NULL));
```

Comment:
```suggestion
            TempMDTuple temp_n = MDNode::getTemporary(Lh->getContext(), None);
```

Reply:
Sure, I wanted to keep the diff minimal but that's better.

Reply:
This does not seem equivalent. I reverted the change.